### PR TITLE
Add skinny banner to code.org/student/middle-high for AFE scholarships

### PR DESCRIPTION
**Pull request by kelbyhawn**
Add an Amazon Future Engineer skinny banner to https://code.org/student/middle-high
- Needed to refactor the `middle-high.haml` file to include the `responsive_full_width` theme
  - Added wrapper divs around the content in each section, 
  - Deprecated the `student/school-styles.scss` stylesheet
  - Some whitespace changes
- Shows for English only

**Jira ticket:** [ACQ-1170](https://codedotorg.atlassian.net/browse/ACQ-1170)

----

## Desktop
Desktop

## Tablet
Screenshot 2023-11-14 at 1 37 35 PM

## Mobile
Screenshot 2023-11-14 at 1 38 18 PM

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/student/middle-high.haml [hunk 1]
```diff
@@ -1,185 +1,196 @@
...
+= view :afe_scholarship_skinny_banner
+
+%section
+  .wrapper
+    %h2
+      =hoc_s(:middle_high_page_hero_title)
+    %p=hoc_s(:middle_high_page_hero_description)
+    .action-block.action-block--two-col
+      .media-wrapper.col-50
+        %img{src: "/images/houreofcode-students-banner.png"}
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/student/middle-high.haml]: Nit: would you mind adding an altText tag here please

File: pegasus/sites.v3/code.org/public/student/middle-high.haml [hunk 2]
```diff
@@ -1,185 +1,196 @@
...
+= view :section_divider_line
+
+%section
+  .wrapper
+    %h2
+      =hoc_s(:middle_high_page_additional_resources_title)
+    %p=hoc_s(:middle_high_page_additional_resources_description)
+    .action-block.action-block--two-col{style: "margin-bottom: 2rem"}
+      .media-wrapper.col-50
+        %img{src: "/images/video-library-banner.png"}
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/student/middle-high.haml]: Nit: same here please

File: pegasus/sites.v3/code.org/public/student/middle-high.haml [hunk 3]
```diff
@@ -1,185 +1,196 @@
...
-    .action-block.action-block--one-col.flex-space-between
-      .content-wrapper
+          =hoc_s(:middle_high_page_video_library_title)
+        %p.heading-sm
+          =hoc_s(:middle_high_page_video_library_description)
+        %a.link-button{href: "https://code.org/videos"}
+          =hoc_s(:middle_high_page_video_library_link)
+    .action-block.action-block--two-col
+      .media-wrapper.col-50
+        %img{src: "/images/cs-journeys-banner.png"}
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/student/middle-high.haml]: Nit: same here please

File: pegasus/sites.v3/code.org/public/student/middle-high.haml [hunk 4]
```diff
@@ -1,185 +1,196 @@
...
+          %img{src: "/images/courses-catalog/CSP.png", alt: ""}
+          %p
+            =hoc_s(:middle_high_page_cs_card_description_2)
+        .content-footer
+          %a.link-button{href: "https://code.org/csp", target: "_blank", rel: "noopener noreferrer"}
+            =hoc_s(:middle_high_page_cs_link_2)
+      .action-block.action-block--one-col.flex-space-between
+        .content-wrapper
+          %p.overline
+            grades
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/student/middle-high.haml]: Would it be possible to use a translatable string for "grades" in the few places it's explicitly typed out in this file?

**Review by TurnerRiley — APPROVED** [review]
Looks good! A few small nits

File: pegasus/sites.v3/code.org/public/student/middle-high.haml (hunk 4 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/student/middle-high.haml]: Ah yes, definitely! I missed this when reviewing Dmytro's original PR 🤦‍♀️

**Pull request merged**